kaldb/src/main/java/com/slack/kaldb/logstore/search/KaldbDistributedQueryService.java

```diff
@@ -48,13 +58,13 @@
       // take
       // Alternately use completeOnTimeout and the value can then we configured per request and not
       // as part of the config
-
-      KaldbServiceGrpc.KaldbServiceFutureStub stub =
-          Clients.newClient(server, KaldbServiceGrpc.KaldbServiceFutureStub.class)
-              .withInterceptors(
-                  GrpcTracing.newBuilder(Tracing.current()).build().newClientInterceptor())
-              .withDeadlineAfter(READ_TIMEOUT_MS, TimeUnit.MILLISECONDS);
-      queryServers.add(ListenableFuturesExtra.toCompletableFuture(stub.search(request)));
+      queryServers.add(
+          ListenableFuturesExtra.toCompletableFuture(
+                  stubLoadingCache
+                      .getUnchecked(server)
```

Comment:
curious why we use `getUnchecked` and not `get` ?

Reply:
Per the doc block, `getUnchecked` should be used when checked exceptions are not thrown by the cache loader. Since `getKaldbServiceGrpcClient` doesn't throw a checked exception this should be safe for us to use.